tests/python/test_xgboost_integration.py

```diff
@@ -130,3 +130,94 @@ def test_nonlinear_objective(tmpdir, objective, max_label, global_bias, toolchai
     out_pred = predictor.predict(batch)
     expected_pred = bst.predict(dtrain)
     np.testing.assert_almost_equal(out_pred, expected_pred, decimal=5)
+
+
+@pytest.mark.skipif(not has_sklearn(), reason='Needs scikit-learn')
+@pytest.mark.parametrize('toolchain', os_compatible_toolchains())
+def test_xgb_deserializers(tmpdir, toolchain):
+    # pylint: disable=too-many-locals
+    """Test Boston data (regression)"""
+    from sklearn.datasets import load_boston
+    from sklearn.model_selection import train_test_split
+
+    # Train xgboost model
+    X, y = load_boston(return_X_y=True)
+    X_train, X_test, y_train, y_test = train_test_split(
+        X, y, test_size=0.2, shuffle=False
+    )
+    dtrain = xgboost.DMatrix(X_train, label=y_train)
+    dtest = xgboost.DMatrix(X_test, label=y_test)
+    param = {'max_depth': 8, 'eta': 1, 'silent': 1, 'objective': 'reg:linear'}
+    num_round = 10
+    bst = xgboost.train(param, dtrain, num_boost_round=num_round,
+                        evals=[(dtrain, 'train'), (dtest, 'test')])
+
+    # Serialize xgboost model
+    model_bin_path = os.path.join(tmpdir, 'serialized.model')
+    bst.save_model(model_bin_path)
```

Comment:
This line is currently crashing in the CI: https://travis-ci.org/github/dmlc/treelite/jobs/731392804

Reply:
Interesting. I wasn't able to reproduce this locally, and I don't think I touched anything that would have interfered with xgboost's model dump capabilities, but I'll see what I can find tomorrow.

Reply:
I was able to reproduce this both on this PR and on mainline. I'll create an issue and try to add a fix in a separate PR before we land this one.